docs/sandbox/DataOverlay.md

```diff
@@ -0,0 +1,66 @@
+# Data Overlay
+TODO DataOverlay
+
+
+## Contact Info
+
+
+## Changelog
+- Initial version (December 2021)
+
+
+## Documentation
+TODO
```

Comment:
Copy in some of the text you wrote on the user mailing list and in the issue. Just to give people an idea of what this can be used for and a hint on how to use it.